commit 7d07c8e32153f5196725a6380b7534e10ea36e49
Author: jedlee <jedlee616@gmail.com>
Date:   Tue Feb 17 21:21:33 2026 +0800

    Support the "supertonic-tts-mnn" model in Android MnnLImChat App.

diff --git a/apps/Android/MnnLlmChat/app/src/main/assets/model_market.json b/apps/Android/MnnLlmChat/app/src/main/assets/model_market.json
index 0d72ddb0..9515a1fc 100644
--- a/apps/Android/MnnLlmChat/app/src/main/assets/model_market.json
+++ b/apps/Android/MnnLlmChat/app/src/main/assets/model_market.json
@@ -1730,7 +1730,7 @@
         {
             "modelName": "bert-vits2-MNN",
             "tags": [
-                "TTS"
+                "TTS", "English and Chinese"
             ],
             "categories": [],
             "size_gb": 1.5,
@@ -1740,6 +1740,20 @@
                 "HuggingFace": "taobao-mnn/bert-vits2-MNN"
             },
             "file_size": 1392023806
+        },
+        {
+            "modelName": "supertonic-tts-mnn",
+            "tags": [
+                "TTS", "English"
+            ],
+            "categories": [],
+            "size_gb": 0.43,
+            "sources": {
+                "ModelScope": "MNN/supertonic-tts-mnn",
+                "Modelers": "MNN/supertonic-tts-mnn",
+                "HuggingFace": "taobao-mnn/supertonic-tts-mnn"
+            },
+            "file_size": 464403825
         }
     ],
     "asr_models": [
